staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -1344,6 +1344,9 @@ const (
 // TODO: pass structured data to these actions, and document that data here.
 type Handler struct {
 	// One and only one of the following should be specified.
+	// GuestAgentPing specifies the action to ping the guest inside the launcher for readiness probe
+	// +optional
+	GuestAgentPing *GuestAgentPing `json:"guestAgentPing,omitempty"`
```

Comment:
Could you move this lower so that the `// One and only one of the following should be specified.` remains on the Exec probe which is the first one alphabetically and therefore shows in the docs as first?
Or we find a better way to put this comment.

Reply:
Changed